### ISSUE TITLE
Cancel button on terms page doesn't do anything

### ISSUE THREAD

**Issue by simonpoole** [openstreetmap/openstreetmap-website#2514]
Probably an error I built in, will investigate.

**Comment by simonpoole** [comment]
OK "not my fault". The button should open a wiki page but hits a CSP issue

terms:1 Refused to send form data to 'https://wiki.openstreetmap.org/' because it violates the following Content Security Policy directive: "form-action 'self'".

**Comment by tomhughes** [comment]
Well the real issue that it's form button when it doesn't need to be... We just want to do a GET without any form data then there wouldn't be a problem... We just need to work out how to do that.

**Comment by scottrobel** [comment]
![girl](https://user-images.githubusercontent.com/54362821/73573226-db88e900-4440-11ea-9170-d6fb3bb54c32.png)
hi i love this project! i can't wait to get a job!

**Comment by openbrian** [comment]
It should be pretty easy to style an anchor tag to look like a button.

**Comment by gravitystorm** [comment]
> It should be pretty easy to style an anchor tag to look like a button.

Yes, we're moving to bootstrap for forms and buttons, and they have both buttons that look like links, and links that look like buttons, so we'll be sorted. 

https://getbootstrap.com/docs/4.5/components/buttons/

**Comment by openbrian** [comment]
I just happen to be using this which is working now.  It's a blue button with white text.  It looks pretty good.

```ruby
<%= button_to t(".edit"), edit_thing_path(thing), :method => :get %>
```

It comes out like this

```html
<form method=GET><input type=submit value="Edit"></form>
```
